tox.ini

```diff
@@ -10,20 +10,20 @@ envlist =
     build_docs
 
 [testenv]
-pypi_filter = https://raw.githubusercontent.com/sunpy/sunpy/main/.test_package_pins.txt
 # We use bash in some of our environments so we have to whitelist it.
```

Comment:
Same with this, if we move it to the top maybe git will be ok with future changes?

We could look into adding this into the base template?